client-v2/src/components/FrontsEdit/CollectionComponents/CollectionItem.tsx

```diff
@@ -161,7 +161,9 @@ const mapDispatchToProps = (dispatch: Dispatch) => {
         dispatch(
           insertArticleFragment(
             { id: 'clipboard', type: 'clipboard', index: 0 },
-            id,
+            // @TODO - if this is proving too slow we can pass the whole external article into
```

Comment:
Does this @todo still apply? (looking at transformExternalArticle etc. below)

Reply:
Yeah it does, currently we don't have the external article in the scope of a`CollectionItem`. So either I could add another `MappableDropType` (maybe not the nicest name). So `CAPI`, `REF` and now `CAPI_ARTICLE_ID_THAT_IS_ALREADY_FETCHED_AND_IN_EXTERNAL)ARTICLES_I_PROMISEEEE` that would mean we could just pass a CAPI id and assume it didn't need fetching (bit dangerous?). Or otherwise pass in the whole external article to the `COLLECTION_ITEM` and change `REF` to `CAPI`. Does that make _any_ sense?